modules/test/scripts/init-couchbase/init-buckets.sh

```diff
@@ -13,24 +13,24 @@ CB_BUCKET_RAMSIZE="${CB_BUCKET_RAMSIZE:-128}"
 
 echo "couchbase-cli bucket-create travel-sample..."
 /opt/couchbase/bin/couchbase-cli bucket-create \
--c localhost -u ${CB_USER} -p ${CB_PSWD} \
---bucket travel-sample \
---bucket-type couchbase \
---bucket-ramsize ${CB_BUCKET_RAMSIZE} \
---bucket-replica 0 \
---bucket-priority low \
---bucket-eviction-policy fullEviction \
---enable-flush 1 \
---enable-index-replica 0 \
---wait
+    -c localhost -u ${CB_USER} -p ${CB_PSWD} \
```

Comment:
ah good, you/vscode have added that indentation where it was distressing me last time :)

Reply:
Yeah it was annoying me too, so I changed my plugin for formatting shell scripts: https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format

Seems much better!